app/assets/javascripts/discourse/app/widgets/default-notification-item.js

```diff
@@ -106,7 +106,10 @@ export const DefaultNotificationItem = createWidget(
     },
 
     icon(notificationName) {
-      let icon = iconNode(`notification.${notificationName}`);
+      return iconNode(`notification.${notificationName}`);
+    },
+
+    _add_a11y_attrs_to(icon, notificationName) {
```

Comment:
That's not a very JS name 😛 

```suggestion
    _addA11yAttrsTo(icon, notificationName) {
```